python/nano/notebooks/pytorch/tutorial/pytorch_inference_onnx.ipynb

```diff
@@ -11,20 +18,54 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Environment Preparation\n",
-    "Before you start with APIs delivered by bigdl-nano, you have to make sure BigDL-Nano is correctly installed for PyTorch. If not, please follow [this](../../../../../docs/readthedocs/source/doc/Nano/Overview/nano.md) to set up your environment.<br><br>\n",
-    "To use onnxruntime accelerator, you need to install some onnx packages as follows to set up your environment with ONNXRuntime acceleration.\n",
-    "```bash\n",
-    "pip install onnx onnxruntime\n",
-    "```"
+    "### Introduction\n",
+    "In this notebook we will describe how to apply ONNXRuntime Acceleration on inference pipeline with the APIs delivered by BigDL-Nano in 4 simple steps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 0: Prepare Environment\n",
+    "Before you start with APIs delivered by bigdl-nano, you have to make sure BigDL-Nano is correctly installed for PyTorch.\n",
+    "\n",
+    "We recommend to run below commands, especially `source bigdl-nano-init` before jupyter kernel is started, or some of the optimizations may not take effect."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# nightly bulit version\n",
+    "!pip install --pre --upgrade bigdl-nano[pytorch]\n",
+    "# set env variables\n",
+    "!source bigdl-nano-init"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Step 0: Load Data\n",
-    "Here we use two images from the dog_vs_cat dataset from kaggle. You can get the images from [here](https://www.kaggle.com/c/dogs-vs-cats)"
```

Comment:
@yangw1234 Not sure why we wrote dog_vs_cat dataset here, it seems that in the code we are using [Oxford-IIIT Pet Dataset](https://www.robots.ox.ac.uk/~vgg/data/pets/). Please have a look whether this correction of description is reasonable, thank you!

Reply:
This is an error. Your correction is perfect.